packit_service/worker/whitelist.py

```diff
@@ -101,12 +101,11 @@ def get_account(self, account_name: str) -> Optional[dict]:
 
     def add_account(self, github_app: InstallationEvent) -> bool:
         """
-        Add account to whitelist, if automatic verification of user
-        (check if user is packager in fedora) fails, account is still inserted in whitelist
-         with status : `waiting`.
-         Then a scripts in files/scripts have to be executed for manual approval
+        Add account to whitelist.
+        Status is set to approved_automatically' or 'waiting' (in case the account is not in FAS).
```

Comment:
```suggestion
        Status is set to 'waiting' or to 'approved_automatically' if the account is a packager in Fedora.
```

- comment follows the actual order things are going to happen in the code (the reader doesn't have to make the mental inversion).
- the condition in the context of this method is "is packager in Fedora", FAS is an implementation detail of `_is_packager`.